package.json

```diff
@@ -20,6 +20,7 @@
     "postinstall": "patch-package",
     "start": "craco start",
     "build": "craco build",
+    "analyze": "source-map-explorer 'build/static/js/*.js'",
```

Comment:
analyze seems a bit vague?

Reply:
`analyze` is the command used as part of CRA, so it's idiomatic here.

Reply:
correct! it's not present by default, but docs use that name there too https://create-react-app.dev/docs/analyzing-the-bundle-size/